### ISSUE TITLE
Footer logo clickable/active state

### ISSUE THREAD

**Issue by BogdanDogaru** [bbc/simorgh#349]
**Acceptance Criteria**
> Given I view the footer
> When I hover on the brand logo
> Then brand logo must have a visible state change
> And it must not be identified by colour alone

> Given I view the footer
> When I navigate by keyboard
> And I focus on the brand logo
> Then brand logo must have a visible state change
> And it must not be identified by colour alone
> And in addition it must have browser default styling (i.e. outline around brand logo)

**Describe the solution you'd like**
We need to style the logo in the Footer component to match the UX&D designs referenced in https://jira.dev.bbc.co.uk/browse/NEWSART-595

Note: should be similar to this Header ticket and reuse the component: https://github.com/bbc/simorgh/issues/346

**Comment by sareh** [comment]
Closed as duplicate. (https://github.com/bbc/simorgh/issues/226) ACs are listed on that ticket.
